src/index.js

```diff
@@ -1,6 +1,9 @@
+// @flow
+import React from 'react';
 import 'capture-core-utils/extensions/asyncForEachArray';
 import 'capture-core-utils/extensions/arrayToHashMap';
-import './locales';
```

Comment:
Would be curious to see how other applications are solving this part.

Reply:
I tested without including this locales import and it didn't work, so I'm leaving it here.